docs/install.html

```diff
@@ -101,7 +102,7 @@ <h2><a name="installation"></a>Installation</h2>
 		already installed. It's recommended to use the package manager of your OS-distribution for
 		each individual library or, if it doesn't provide those, that of PHP itself called
 		<strong>PEAR</strong>. At least PEAR should most likely be available using the package
-		manager of your OS-distribution. With e.g. a Debian based Linux simply issue the following
+		manager of your OS-distribution. With e.g., a Debian based Linux simply issue the following
```

Comment:
* I'd simply write OS, not OS-distribution (which isn't English anyway)
* Isn't it the over way around? Debian is based on Linux, not Linux on Debian. Debian makes the OS while Linux being the kernel of it.